docs/cn/index.md

```diff
@@ -58,8 +58,7 @@ InputStream和OutputStream的接口和对内存映射I/O的高效支持。我们
 不同的底层存储系统。目前我们支持Microsoft Azure Blob Store，Amazon S3，Google Cloud Storage，OpenStack Swift，GlusterFS，
 HDFS，MaprFS，Ceph，NFS，Alibaba OSS，Minio以及单节点本地文件系统，后续也会支持很多其它的文件系统。
 
-* **[Alluxio存储](Alluxio-Storage.html)** Alluxio可以管理内存和本地存储如SSD
-和HDD,以加速数据访问。如果需要更细粒度的控制，分层存储功能可以用于自动管理不同层之间的数据，保证热数据在更快的存储层上。
+* **[Alluxio存储](Alluxio-Storage.html)** Alluxio 可以管理内存和本地存储如 SSD 和 HDD，以加速数据访问。如果需要更细粒度的控制，分层存储功能可以用于自动管理不同层之间的数据，保证热数据在更快的存储层上。
```

Comment:
In our convention, we do not add space between English words and Chinese words. Thanks for your contribution!

Reply:
I pushed a new commit, please check the correctness